docker-for-azure/upgrade.md

```diff
@@ -42,6 +42,18 @@ $ docker run \
   docker4x/upgrade-azure:17.06.0-ce-azure1
 ```
 
+If you are already on a version more recent than 17.06.1 CE or 17.07.0 CE, you can use the below script to initiate the upgrade:
+
+```bash
+$ upgrade.sh YY.MM.X-ce-azureC
+```
+
+For example, to upgrade from 17.06.1 CE edge to 17.07.0 CE edge, you can use:
```

Comment:
Updated the note. Since `upgrade.sh` is in /usr/bin which is set in PATH, `ash` will always be able to locate it. The full path is not necessary.